### PR TITLE
Enablement for chef -> ansible transition

### PR DESCRIPTION
**Pull request by zmc**
This doesn't move us over yet, but adds all of the (teuthology-side) tweaks we need to make the real transition.

### CODE REVIEW

**Comment by zmc** [comment]
test this please

File: teuthology/task/ansible.py [hunk 1]
```diff
@@ -213,9 +224,13 @@ def _build_args(self):
         """
         Assemble the list of args to be executed
         """
-        fqdns = [r.name.split('@')[-1] for r in self.cluster.remotes.keys()]
+        fqdns = [r.hostname for r in self.cluster.remotes.keys()]
+        # Assume all remotes use the same username
+        user = self.cluster.remotes.keys()[0].user
```
Comment (andrewschoen) [inline teuthology/task/ansible.py]: Just a note: we should add the teuthology_user to teuthology.config sometime. We'll eventually need that.

Comment (zmc) [inline teuthology/task/ansible.py]: I actually think we might want to get the user from paddles on a per-host basis. I'm not positive though.

Comment (andrewschoen) [inline teuthology/task/ansible.py]: Yeah, maybe so. Either way, not an issue for this PR :)

**Pull request merged**